drafts/web_dev_tools.md

```diff
@@ -0,0 +1,147 @@
+VS Code allows to apply a theme to text syntax highlighting in the editor based on preferences. There are many themes in the VS Marketplace that are free.
+
+###### Command line
+VS Code comes with an integrated command-line terminal. On Windows, the terminal shows up as the Command Prompt. On Mac and Linux it shows up as Bash prompt.
+
+###### Go to definition
+While programming or scripting, it can often happen that we face ourselves with a variable or method that we don't know. In that case this feature can come handy, because it makes possible to jump right to the definition of it.
+
+### Sublime Text
+Another editor, the Sublime Text, is a good choice for beginners. It is an all in one text editor for code and markup. As key features we can mention the easy switch between files and functions, the Javascript and Typescript support, the Python API, the cross-platform functionality and many more.
+
+#### Main features
+
+###### Autocompletion, Syntax Highlight, Code Folding
+Sublime has native support for a lot of programming languages and can color elements appropriately. The code editor is also able to warn of parsing errors in real time. Code Folding makes large pieces of code more compact to view by hiding parts of it.
+
+###### Customizability
+Sublime Text can be fully customized via plugins. The integrated Package Manager allows us to find, install, upgrade and remove plugins directly from inside the editor, usually without restart. We can download packages from Github, Bitbucket and PackageControl.io. PackageControl.io is Sublime's own repository, where the community behind the code editor publishes loads of plugins to enhance its capabilities. There are more than 3800 packages available.
+
+###### Lightweight, Fast and Stable
+It starts fast and crashes rarely. What else do we need?
+
+###### Powerful search
+Fast and powerful search is also an important feature. Its regular search and replace function can look for regular expressions, make sensitive searches or look for whole words. Plus we can add search results to buffer for later use.
+
+###### Simultaneous editing
+One of the most interesting features of Sublime Text is the ability to use several cursors at the same time. That means it is possible to make the same edit in several places at the same time. The same feature can be used to reformat several lines of code at the same time.
+
+###### Minimap
+It is a small file overview in the upper right corner. It is useful to get an idea of how large the open file is and also shows the current position while moving the scrollbar. 
+
+###### Screens and panels
+It has the ability to display text files in different ways. First we can have several files ordered in tabs like in a browser window. That's the standard way. More than that, we can display up to four files at once if form of a grid. Plus Sublime Text comes with other layout options like columns and rows as well as fullscreen and distraction free editing modes.
```

Comment:
```suggestion
It can display text files in different ways. First, we can have several files ordered in tabs like in a browser window. That's the standard way. More than that, we can display up to four files at once in the form of a grid. Plus Sublime Text comes with other layout options like columns and rows as well as fullscreen and distraction-free editing modes.
```